libs/chlorophyll/scss/components/form/checkbox/checkbox.stories.mdx

```diff
@@ -203,6 +215,7 @@ Group form controls such as checkboxes and radio buttons using `<fieldset>` and
         <span id="checkboxGroupHelp1" class="form-info">
           Checkbox group description
         </span>
+        <div>
         <label class="form-control">
```

Comment:
One small thing: indentation inside the div

Reply:
https://github.com/sebgroup/green/pull/658/commits/475b5b335505ab620571948371b3ef32dace523d